<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgConnectionSettings</class>
 <widget class="QDialog" name="DlgConnectionSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="2,0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>239</red>
             <green>41</green>
             <blue>41</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>239</red>
             <green>41</green>
             <blue>41</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Do not tinker with these values unless you know what you're doing!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="1">
        <widget class="QSpinBox" name="sbTimeout">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leUrl">
         <property name="placeholderText">
          <string>without http://</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Server's API base URL:</string>
         </property>
         <property name="margin">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="cbUrl">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="cbPubKey">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lePubKey">
         <property name="placeholderText">
          <string>Base64-encoded</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Server's public key:</string>
         </property>
         <property name="margin">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="cbTimeout">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Request timeout in seconds:</string>
         </property>
         <property name="margin">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOkay">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgConnectionSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOkay</sender>
   <signal>clicked()</signal>
   <receiver>DlgConnectionSettings</receiver>
   <slot>onBtnOkayClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbUrl</sender>
   <signal>toggled(bool)</signal>
   <receiver>DlgConnectionSettings</receiver>
   <slot>onCheckboxToggled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbPubKey</sender>
   <signal>toggled(bool)</signal>
   <receiver>DlgConnectionSettings</receiver>
   <slot>onCheckboxToggled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbTimeout</sender>
   <signal>toggled(bool)</signal>
   <receiver>DlgConnectionSettings</receiver>
   <slot>onCheckboxToggled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCheckboxToggled()</slot>
  <slot>onBtnOkayClicked()</slot>
 </slots>
</ui>
